CMakeLists.txt

```diff
@@ -17,6 +17,42 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+# Taken from https://gist.github.com/steakknife/c36c99b51703fc6f6c1b
+macro(find_package_prefer_brew _package)
+  unset(_brew_path)
+  unset(_brew_pkg_lower)
+  unset(_has_brew)
+  find_program(_has_brew NAMES brew DOC "path to Homebrew executable")
+  if(_has_brew)
+    string(TOLOWER ${_package} _brew_pkg_lower)
+    execute_process(COMMAND brew --prefix ${_brew_pkg_lower} ERROR_QUIET OUTPUT_VARIABLE _brew_path)
```

Comment:
We could set the option `OUTPUT_STRIP_TRAILING_WHITESPACE` and avoid the regex replace done few lines later.